src/components/ReviewList.jsx

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+function ReviewList(newReviewID) {
+  const [reviews, setReviews] = useState([]);
+  useEffect(() => {
+    axios
+      .get("https://nc-games-m65q.onrender.com/api/reviews")
+      .then((response) => {
```

Comment:
This works, but would be nicer if you could extract this functionality into an api.js file and use a function that's exported from there that does the same. Have a look at the Axios section of the notes 👍